src/app.js

```diff
@@ -324,6 +374,24 @@ function App() {
       document.removeEventListener('contentEmpty', contentEmpty);
       document.removeEventListener('contentNotEmpty', contentNotEmpty);
 
+      // Clear out our wake lock as well as possible.
+      if (
+        wakeLockHandle.wakeLock !== null &&
+        wakeLockHandle.wakeLock.released !== true
+      ) {
+        wakeLockHandle.wakeLock
+          .release()
```

Comment:
```suggestion
      // Clear out our wake lock as well as possible.
      if (
        wakeLockHandle.sentinel !== null &&
        wakeLockHandle.sentinel.released !== true
      ) {
        wakeLockHandle.sentinel
          .release()
```

I am a bit confused here.

This refers to `wakeLockHandle.wakeLock` but as I see it `preventSleep()` will return an object with a `sentinel` attribute. Could it be something which started out as `sentinel` but was renamed `sentinel` later on?

Reply:
Yes, looks like an error introduced during renaming. Good catch, thanks!

Reply:
Yes, looks like an error introduced during renaming. Good catch, thanks!